cmd/loki-canary/main.go

```diff
@@ -148,7 +182,7 @@ func main() {
 	}()
 
 	terminate := make(chan os.Signal, 1)
-	signal.Notify(terminate, syscall.SIGTERM, os.Interrupt)
+	signal.Notify(terminate, syscall.SIGTERM, syscall.SIGINT, os.Interrupt)
```

Comment:
Why was this added?

Reply:
Oops. `os.Interrupt` is same as `syscall.SIGINT`. Overlooked. will remove